.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What about https://github.com/WordPress/gutenberg/blob/master/babel.config.js?

Reply:
For some reasons `parcel` didn't pick it up. It wanted a `.babelrc` or a `babel` entry in `package.json`

Reply:
`wp-scripts build/start` no longer needs this file, so you can remove the other one. It will pick this one as well.